CHANGES.rst

```diff
@@ -1,6 +1,23 @@
 Changes
 =======
 
+Version 0.5.0 (2019-04-23)
+--------------------------
+
+- Enables to create workflows on demand.
+- Add support for interactive sessions to manage workflows directly with interactive tools such as Jupyter.
```

Comment:
What about: "- Adds support to create interactive sessions so the workspace can be explored and modified through a Jupyter notebook."?